packages/migration-graph-ember/test/utils/discover-ember-packages.test.ts

```diff
@@ -1,8 +1,37 @@
-import { describe, expect, test } from 'vitest';
+import { fileURLToPath } from 'node:url';
+import { dirname, resolve } from 'node:path';
+import { describe, expect, test, beforeEach, afterEach } from 'vitest';
+import { Project } from 'fixturify-project';
+import { EmberAppProjectGraph } from '../../src/index.js';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+const FIXTURE_APP_PATH = resolve(__dirname, '../fixtures/monorepo');
 
-// TODO: should have a test
 describe('discover-ember-packages', () => {
-  test.todo('should have test', () => {
-    expect(true).toBe(false);
+  let projectGraph: EmberAppProjectGraph;
+  let project: Project;
+  beforeEach(async () => {
+    project = Project.fromDir(FIXTURE_APP_PATH, { linkDeps: true, linkDevDeps: true });
+    project.addDevDependency('ember-source');
+    await project.write();
+    projectGraph = new EmberAppProjectGraph(project.baseDir);
+  });
+
+  afterEach(() => {
+    project.dispose();
+  });
+
+  test('returns the graph in top sort order', () => {
+    projectGraph.discover();
+    const nodes = projectGraph.graph.topSort();
+
+    expect(nodes.map((n) => n.content.pkg?.packageName)).toEqual([
+      '@company/b',
+      '@company/a',
+      '@company/c',
+      '@company/d',
+      'monorepo',
+    ]);
```

Comment:
This should be added to [`EmberAppProjectGraph-test.ts`](https://github.com/rehearsal-js/rehearsal-js/blob/master/packages/migration-graph-ember/test/entities/ember-app-project-graph.test.ts)

As it's largely asserting that the order was correct given a project fixture.

This test could stay here if we're calling discoverEmberPackages explicitly and asserting it's output from the method.